app/src/main/java/com/example/myapplication/PitFragment.java

```diff
@@ -36,9 +36,11 @@ public class PitFragment extends Fragment {
     UserModel userModel;
 
     public PitFragment() {
-        FragmentActivity fragmentActivity = requireActivity();
-        ViewModelProvider viewModelProvider = new ViewModelProvider(fragmentActivity);
-        userModel = viewModelProvider.get(UserModel.class);
+        try {
+            FragmentActivity fragmentActivity = getActivity();
+            ViewModelProvider viewModelProvider = new ViewModelProvider(fragmentActivity);
+            userModel = viewModelProvider.get(UserModel.class);
+        } catch(Exception e){}
```

Comment:
Why do you need to catch exceptions? This usually indicate there's something fishy going on...

Reply:
oh because camila needs to change this i told her how to keep going with that but rn it throws an excepion